src/screens/UsernamePasswordLoginViewModel.ts

```diff
@@ -50,9 +50,11 @@ const useUsernamePasswordLoginViewModel = () => {
 		// see {@link Registration.serverUrl}. If no server base URL is provided, then the base URL
 		// defined in {@link Configuration.baseUrl} will be used.
 		const client = ClientProvider.getInstance().client;
+		const deviceInformation =
+			(await client?.localData.deviceInformation()) ?? DeviceInformationUtils.create();
 		await client?.operations.registration
 			.username(usernameToRegister)
-			.deviceInformation(DeviceInformationUtils.create())
+			.deviceInformation(deviceInformation)
```

Comment:
❓What is the reason for this change?

Reply:
We discussed it earlier and it's a common mistake to do not reuse the already existing device information.
On the other hand if you do not reuse the already existing device information and start to use the new `devices` endpoint to check device information mismatches you will get a `DeviceInformationMismatch` if multiple accounts are registered.